pkg/registry/policy/podsecuritypolicy/storage/storage.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/printers"
 	printersinternal "k8s.io/kubernetes/pkg/printers/internalversion"
 	printerstorage "k8s.io/kubernetes/pkg/printers/storage"
-	"k8s.io/kubernetes/pkg/registry/extensions/podsecuritypolicy"
+	"k8s.io/kubernetes/pkg/registry/policy/podsecuritypolicy"
 )
 
 // REST implements a RESTStorage for PodSecurityPolicies.
```

Comment:
were we going to change `DefaultQualifiedResource: extensions.Resource("podsecuritypolicies"),` at this point?

Reply:
Do you know what `DefaultQualifiedResource` does? What role it plays and what will be impact of this change? Last time I changed it, I introduced strange issue that only @sttts was able to debug.